### PR TITLE
Story4 task1

### PR DESCRIPTION
**Pull request by west-country**
Description

    - when first keystroke is registered start overall 60 second timer
    - also, display a timer that counts down using interval?
    - other functionality runs whilst the timer is running

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+const resultsPopup = document.querySelector("#resultsPopup")
+let countdown = 59
+const once = {  //option for addEventListener
+    once: true
+}
+
+textInput.addEventListener('keyup', () => {
+    const countdownForDisplay = setInterval(()=>{
+        timer.innerHTML = countdown
+        countdown--
```
Comment (robin-hodge) [inline script.js]: this needs a space?

File: script.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    }, 1000)
+    const countdownForInput = setTimeout(() => {
+        textInput.disabled = true
```
Comment (robin-hodge) [inline script.js:91]: We have styled out the input altoghter, and the timer, at the end - this perhaps leaves space for a resset button 🤞

File: script.js (hunk 1 above)
Comment (melody-sylvestre) [reply script.js]: done

File: script.js (hunk 2 above)
Comment (melody-sylvestre) [reply script.js:91]: we'll see....

**Pull request merged**